### PR TITLE
add help button in home page

### LINKED ISSUE #2281: [FEATURE] Help button to link to GitHub Discussions
### Description

Add a help icon and link to GitHub Discussions

Screenshot 2022-11-26 at 12 09 16

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by joeshajan**
## Fixes Issue

Closes #2281

## Changes proposed

Added a new help button on the index page. It links to discussions in linkfree repo

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2283"></a>

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -103,8 +106,20 @@ export default function Home({ data }) {
             height="638"
           />
         </div>
+        <a
+          href="https://github.com/EddieHubCommunity/LinkFree/discussions"
+          legacyBehavior
```
Comment (eddiejaoude) [inline pages/index.js:111]: Looks good, maybe we need it to open in a new tab?

Comment (joeshajan) [reply pages/index.js]: Correct @eddiejaoude I did that please take a look

**Review by schmelto — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Comment by eddiejaoude** [comment]
Thank you for your contribution 👍

**Comment by joeshajan** [comment]
> Thank you for your contribution 👍

Thanks @eddiejaoude

**Pull request merged**